### ISSUE TITLE
Bug report templates should be semantic

### ISSUE THREAD

**Issue by mgifford** [ckan/ckan#7186]
Pretty trivial issue, but bolding the headings like you do **CKAN version**

Isn't semantic. Headings should be used to break up major elements, so:

## Describe the bug

I don't there are sub-headings here, but perhaps in other templates. 

### Steps to reproduce

This is a pretty simple fix that will also make the content more machine-readable in the future.
